doc/source/install.rst

```diff
@@ -0,0 +1,27 @@
+.. _installation:
+
+Installation
+============
+
+.. attention:: The nomenclature requires python >= 3.8
+
+
+Via Pip
+-------
+.. attention::  The nomenclature package is distributed as "nomenclature-iamc" on pypi.
+
+.. code-block:: bash
+
+    pip install nomenclature-iamc
+
+
+Installing From Source
+----------------------
+
+nomenclature can also be installed from source. This will get you the latest version
+of the main branch.
+
+.. code-block:: bash
+
+    pip install -e git+https://github.com/IAMconsortium/nomenclature@main#egg=nomenclature
```

Comment:
```suggestion
    $ pip install -e git+https://github.com/IAMconsortium/nomenclature@main#egg=nomenclature
```

Reply:
I would not do that as it suggests that the user needs to actually type the dollar sign. See [this question](https://stackoverflow.com/questions/59795354/what-does-the-dollar-sign-represent-in-front-of-lines) on stackoverflow. I would put just the command that the user can copy and paste verbatim into the shell.